openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -1217,3 +1218,26 @@ def _replace_username_for_all_models(self, current_username, new_username, repla
                 new_username,
             )
         return True
+
+
+class DemographicsStatusView(APIView):
```

Comment:
Question that is only marginally related to your PR...

What is the determining factor to use the APIView over a ViewSet?

Reply:
APIView is a version of Viewset with methods and overrides more inline with the HTTP Methods. (i.e. get-> GET, post-> POST. 

I tend to use the API view for API specific changes like this one that is not related to a model.